addons/mail/static/src/discuss/call/common/peer_to_peer.js

```diff
@@ -549,6 +554,9 @@ export class PeerToPeer extends EventTarget {
                     return;
                 }
                 peer.isBuildingAnswer = false;
+                if (!this.isActive) {
+                    return;
+                }
```

Comment:
Shouldn't it be present in the `catch` too, before `_recover`?

Reply:
`_recover` will not do anything if there is no peer (which are cleared on disconnect)